src/java.base/share/data/cacerts/certignaca

```diff
@@ -0,0 +1,29 @@
+Owner: CN=Certigna, O=Dhimyotis, C=FR
+Issuer: CN=Certigna, O=Dhimyotis, C=FR
+Serial number: fedce3010fc948ff
+Valid from: Fri Jun 29 15:13:05 GMT 2007 until: Tue Jun 29 15:13:05 GMT 2027
+Signature algorithm name: SHA1withRSA (weak)
```

Comment:
Don't include "(weak)" in this line. The signature algorithm of root certificates is not a security issue, since the certificate fingerprint has been verified and the public key is trusted directly.